### ISSUE TITLE
How dose the maskrcnn+ be  trained  which mentioned in iSaid paper?

### ISSUE THREAD

**Issue by lyw615** [CAPTAIN-WHU/iSAID_Devkit#13]
After clipping all train dataset into 800*800 tile with 200 overlap, about 28K  tiles are got.  As mentioned in isaid paper`>  Our models are trained on 8 GPUs for 180k iterations with an initial learning rate of 0.025`,  with setting images_per_gpu value 2. I calculate train epochs is about 100.  
Can I get the right training epochs  of maskrcnn+ which mentioned in iSaid paper?

**Comment by akshitac8** [comment]
@lyw615 Number pf epochs depend upon which code are you using and all the changes for maskrcnn+ are mentioned in the paper.

**Comment by lyw615** [comment]
Other hyperparameters are  same as the paper of maskrcnn writted by  Kaiming He?

**Comment by txfs1926** [comment]
Hi, does the config file of maskrcnn+ uploaded to this repo?

**Comment by akshitac8** [comment]
@lyw615 some hyperparameters are changed to make maskrcnn adapt to satellite data which are mentioned in the paper and rest are same as original maskrcnn.

**Comment by lyw615** [comment]
What's this means “As high scale variation exists within aerial images, we use scale augmentations at six  scales (1200,1000,800,600,400)” ？ Does the aumentation method refer to random zoom?  Can you be more specific?

**Comment by akshitac8** [comment]
@lyw615 these are the train input image sizes.

**Comment by lyw615** [comment]
So the trick here is  resizing images into the six scales ,and then put into network training?

**Comment by akshitac8** [comment]
It's the same as maskrcnn or panet config files

**Comment by akshitac8** [comment]
@lyw615 Is the issue solved?

**Comment by lyw615** [comment]
> @lyw615 Is the issue solved?
Thanks ,i'm training model as mentioned above. I hope to be as close as possible to the accuracy in the paper

**Comment by lyw615** [comment]
- 我还是不太清楚isaid论文里面这句：“As high scale variation exists within aerial images, we use scale augmentations at six scales (1200,1000,800,600,400)”。我的理解是对数据进行了多尺度缩放，比如 原始数据是800*800的，然后resize 到400*400或1200*1200。但是maskrcnn不是只能输入固定尺寸的数据吗？ 即网络要求输入800*800的，  数据增强得到1200*1200，但最后不都会被resize 到800*800再拿进去训练吗？ 
- 我不知道我的理解是否正确哈，如果我理解没错的话，那我想请问这样的多尺度缩放是否真的能起到效果？

**Comment by akshitac8** [comment]
@lyw615 Can you please write the issue in English?
